addons/analytic/models/analytic_plan.py

```diff
@@ -108,7 +120,7 @@ def action_view_analytical_accounts(self):
         result = {
             "type": "ir.actions.act_window",
             "res_model": "account.analytic.account",
-            "domain": [('id', 'in', self.account_ids.ids)],
+            "domain": [('id', 'in', self.all_children_account_ids.ids)],
```

Comment:
Let's add the field in `master`, we are changing enough things in stable like that

```suggestion
            "domain": [('id', 'in',(self + self.children_ids).account_ids.ids)],
```

Reply:
Also not that this is only considering direct children. Isn't it possible to have more than one degree of parenthood?